src/DittoMachine.sol

```diff
@@ -326,6 +331,29 @@ contract DittoMachine is ERC1155D, IERC721Receiver, IERC1155Receiver, CloneList,
         return true;
     }
 
+    function bumpSubsidy(address nft, uint id, address ERC20Contract, bool floor, uint128 amount) external {
+        uint protoId = uint(keccak256(abi.encodePacked(
+            nft,
+            id,
+            ERC20Contract,
+            false
+        )));
+        uint cloneId = protoIdToIndexHead[protoId];
+        assembly ("memory-safe") {
+            mstore(0, protoId)
+            mstore(0x20, cloneId)
+            cloneId := keccak256(0, 0x40)
+        }
+
+        SafeTransferLib.safeTransferFrom(
+            ERC20(ERC20Contract),
+            msg.sender,
+            address(this),
+            amount
+        );
+        cloneIdToSubsidy[cloneId] += amount;
+    }
```

Comment:
Just something to consider here, but do we want to allow anyone to increase the subsidy of a clone?

Reply:
@0xbok no i don't see it being unsafe, just wanted to bring it to attention and think about it a bit. But i think it's fine, might even result in some interesting mechanics built on top of it.